scripts/test-acceptance.sh

```diff
@@ -30,10 +30,8 @@ fi
 
 mkdir -p test/logs
 cd test/logs
-export SELENIUM_BROWSER=phantomjs
```

Comment:
I liked that the tests don't pop a browser when run locally. However, I know that @cahrens preferred it to show Chrome so that she could see what it was doing. Maybe this needs to be a setting, or maybe folks can just set SELENIUM_BROWSER themselves locally.